content/docs/user-guide/data.md

```diff
@@ -0,0 +1,128 @@
+# Working with data
+
+<details>
+
+### ⚙️ Expand for setup instructions
+
+If you want to follow along with this tutorial and try MLEM, you can use our
+[example repo](https://github.com/iterative/example-mlem-get-started). You'll
+need to [fork] it first. Then clone it locally:
+
+[fork]: https://docs.github.com/en/get-started/quickstart/fork-a-repo
+
+```cli
+$ git clone <your fork>
+$ cd example-mlem-get-started
+```
+
+Next let's create an isolated virtual environment to cleanly install all the
+requirements (including MLEM) there:
+
+```cli
+$ python3 -m venv .venv
+$ source .venv/bin/activate
+$ pip install -r requirements.txt
+```
+
+</details>
+
+To use data within the MLEM context, so that it is compatible with other operations
+such as [mlem apply](/doc/command-reference/apply), one needs to save it as a
+[MLEM Object](/doc/user-guide/basic-concepts#mlem-objects).
+
+## Saving data with MLEM
+
+One can use MLEM to save existing data using the [mlem.api.save()](/doc/api-reference/save) API.
+
+As an example, we load the well-known IRIS dataset with `sklearn`, and then save parts of it
+with MLEM locally.
+
+```py
+# prepare.py
```

Comment:
1. This comment is a bit weird imo. marking the file of which this is the contents. Suggest to move this "comment" out of the fenced block and instead have this above the block:
> Consider this `prepare.py` file:

2. I would add a shebang instead actually - `#!/usr/bin/env python` (followed by 2 empty lines as is tradition)

Reply:
done